ground/src/main/java/com/google/android/ground/persistence/sync/TileSetDownloadWorkManager.kt

```diff
@@ -29,7 +29,7 @@ constructor(private val workManager: WorkManager, private val localValueStore: L
   override val workerClass: Class<TileSetDownloadWorker>
     get() = TileSetDownloadWorker::class.java
 
-  override fun preferredNetworkType(): NetworkType =
+  override val preferredNetworkType: NetworkType =
```

Comment:
Same thing here. This will be fixed on instantiation. We might want to use a `get()` so that we can still reference this as a property but it will always have the latest value from the localValueStore.

Reply:
also reverted. thanks